README.md

```diff
@@ -1,3 +1,4 @@
+<img src="https://swift.org/assets/images/swift.svg" alt="Swift logo" width="200">
```

Comment:
Any reason not to just specify this in Markdown’s image format?

`![Swift Logo](https://swift.org/assets/images/swift.svg)`